### DISCUSSION TITLE
How to use "Swap Before and After settings"

### DISCUSSION THREAD

**Discussion by mic-ra** [rsjaffe/MIDI2LR#651] [Q&A] [answered]
I am not able to asign "Swap Before and After settings" to a button. There is a keyboard shortcut "**Ctrl + Alt + Shift + Up Arrow**" but I am not able to asign the **Arrow Up** key to a keyboard shortcut in MIDI2LR.

Is there a way I am not seeing? Or is it simply not possible to use "Swap Before and After settings" in MIDI2LR?

**Comment by rsjaffe** [comment, marked as answer]
See https://github.com/rsjaffe/MIDI2LR/wiki/Plugin-Options-Dialog#keyboard-shortcuts for the proper words to use for key presses. So use `cursor up`

**Reply by mic-ra** [reply]
Thank you. I searched for an hour and missed that somehow. 🙈
